### DISCUSSION TITLE
Ariakit roadmap?

### DISCUSSION THREAD

**Discussion by patrikholcak** [ariakit/ariakit#2215] [General]
Hey, we use reakit in our project and are debating switching to ariakit. Is there any general plan when ariakit will be stable or any roadmap?

From https://github.com/ariakit/ariakit/issues/939 it looks like there’s not much missing in terms of examples and there are new bugfix releases fairly regularly.

From the [What is Ariakit?](https://gist.github.com/diegohaz/bc07491aee61a5f2469574b38c5c1aa0) it seems like the migration should be straight-forward but we’re still a bit worried about it being in alpha.

Thank you!

**Comment by diegohaz** [comment]
#1875 will probably be the last significant change before an official release. But there will be more minor breaking changes after that. I don't know precisely when that will happen, but my goal is to release it this first semester. I recommend using Ariakit if you need any feature already implemented there.
